swing-lib/src/test/java/org/triplea/swing/SwingActionTest.java

```diff
@@ -100,9 +101,11 @@ public void testInvokeAndWaitResult_ShouldReturnActionResultWhenCalledOffEdt() t
   @Test
   public void testInvokeAndWaitResult_ShouldReturnActionResultWhenCalledOnEdt() throws Exception {
     SwingUtilities.invokeAndWait(() -> {
-      assertTrue(
-          Interruptibles.await(() -> assertEquals(VALUE, SwingAction.invokeAndWaitResult(() -> VALUE))),
-          "should not be interrupted");
+      try {
+        SwingAction.invokeAndWaitResult(() -> VALUE);
+      } catch (final InterruptedException e) {
+        throw new RuntimeException(e);
+      }
```

Comment:
AssertNotThrows would suit better here

Reply:
There is a problem here, I think the `throw new RuntimeException` is here purely for compiler reasons and the test cares none at all or never expects an interrupted exception to be thrown. The exception does not seem to be the point of the test. Reading the test method and comparing to nearby methods, it looks likes the return value should be verified, but we are calling a void method, so I'm personally not sure what this test was trying to do and how to properly fix it. Considering this code was only moved, it's not the intent of this PR to update this.

Reply:
Well you did alter the behaviour already.
Previously an AssertionError would be thrown if the Thread got interrupted, now a RuntimException is being thrown.
Most IDE Test-Runner tools treat assertion errors as "failed" and all other exceptions as "errored", so changing the thing as I suggested would maintain the original behaviour while using a newer JUnit API.

Reply:
For clarification (just re-read your answer):
I'm pretty sure all the test did previously was ensuring no exception is thrown.
`Iterruptibles#await` does return true if no `InterruptedException` was thrown and false otherwise.
I'm assuming that this test was written before `assertDoesNotThrow` was introduced to the JUnit API.

Reply:
If I'm not mistaken, the previous behavior would never throw an exception and we would be in this code: 
```
    try {
      return new Result<>(true, Optional.ofNullable(supplier.get()));
    } catch (final InterruptedException e) {
      Thread.currentThread().interrupt();
      return new Result<>(false, Optional.empty());
    }
```

I'm looking for the cleanest way to remove the `Interruptibles` only and not change the test; spending too much time here to refactor the test is not worth it.

Reply:
@DanVanAtta 
Have a look at the `Result` parameters: If an Exception is thrown the first parameter is false, true otherwise.
`Interruptibles#await` is different that `Interruptibles#awaitResult`. The former returns false if the passed actions throws an `InterruptedException` because it ultimately just returns this first parameter.
That's why previously `Interruptibles#await` ultimately ensure no `InterruptedException` is being thrown.

Reply:
Just for reference: https://junit.org/junit5/docs/5.3.0/api/org/junit/jupiter/api/Assertions.html#assertDoesNotThrow(org.junit.jupiter.api.function.Executable)